library/marathon.py

```diff
@@ -297,7 +298,7 @@ def main():
         app=dict(required=False),
         app_id=dict(required=False),
         state=dict(required=True, choices=['present', 'absent', 'updated', 'test','get']),
-        force=dict(required=False, type='bool', default=False),
+        force=dict(required=False, type='bool', default=True),
```

Comment:
Nice, thanks for the PR.

I would say that defaulting to False would be what users expect, also because it's the same in the apis (http://thefactory.github.io/marathon-python/marathon.html#module-marathon.client).

Would you mind changing this bit?